Sketch.java

```diff
@@ -1,36 +1,79 @@
 import processing.core.PApplet;
 
 public class Sketch extends PApplet {
-	
+
+  /**
+  * Description:
+  * Write a program that creates a drawing using processing. 
+  * @author: Hosey K
+  */
 	
   /**
    * Called once at the beginning of execution, put your size all in this method
    */
   public void settings() {
 	// put your size call here
-    size(400, 400);
+    size(500, 500);
   }
 
   /** 
    * Called once at the beginning of execution.  Add initial set up
    * values here i.e background, stroke, fill etc.
    */
   public void setup() {
-    background(210, 255, 173);
+    background(207, 244, 250);
   }
 
   /**
    * Called repeatedly, anything drawn to the screen goes here
    */
   public void draw() {
-	  
-	// sample code, delete this stuff
-    stroke(128);
-    line(150, 25, 270, 350);  
 
-    stroke(255);
-    line(50, 125, 70, 50);  
-  }
+	// Drawing grass 
+    fill(163, 230, 115);
+    rect(-5, 445, 510, 75);
+  
+  // Drawing pathway
+    fill(189, 189, 189);
+    quad(400, 500, 385, 460, 295, 460, 280, 500);
   
-  // define other methods down here.
+  // Drawing house base
+    fill(242, 234, 203);
+    rect(80, 220, 350, 240); 
+
+  // Drawing window base
+    fill(0, 0, 0);
+    rect(125, 310, 75, 65); 
+
+  // Drawing window glass
+    fill(255, 255, 255);
+    rect(130, 315, 30, 25);
+
+    fill(255, 255, 255);
+    rect(165, 315, 30, 25);
+
+    fill(255, 255, 255);
+    rect(130, 345, 30, 25);
+
+    fill(255, 255, 255);
+    rect(165, 345, 30, 25);  
+  
+  // Drawing door 
+    fill(156, 98, 17);
+    rect(295, 300, 90, 160);
+
+  // Drawing door knob 
```

Comment:
line up your comments so they are indented in line with the code they are referring to